server/controllers/temporary-accommodation/manage/arrivalsController.ts

```diff
@@ -50,29 +50,39 @@ export default class ArrivalsController {
     return async (req: Request, res: Response) => {
       const { premisesId, roomId, bookingId } = req.params
       const callConfig = extractCallConfig(req)
-
-      const newArrival: NewArrival = {
-        ...req.body,
-        ...DateFormats.dateAndTimeInputsToIsoString(req.body, 'arrivalDate'),
-        ...DateFormats.dateAndTimeInputsToIsoString(req.body, 'expectedDepartureDate'),
-        type: 'CAS3',
-      }
-
       try {
-        await this.arrivalService.createArrival(callConfig, premisesId, bookingId, newArrival)
+        const error = new Error()
```

Comment:
if we need to assign the error to a `const` why not do it closer to where it will be used?